source/style-guide/style/version.txt

```diff
@@ -0,0 +1,94 @@
+document, we should not reference releases of products that we no longer support.
+That is, when a release of MongoDB Server reaches EOL,  references to this
+release should be removed.
+
+* Users don't need to know a feature was introduced in a release that is EOL,
+  since it suggests a text that is out of date.
+
+* Users don't need to know about features that were removed in a release 
+  that is EOL, since it means the feature is unavailable in all supported 
+  releases.
+
+* Users don't need to know about features that were changed in a release
+  that is EOL, since the change now applies to all available releases.
+
+This topic provides guidelines on how to reference product versions in order to
+make it easier to find these references once the release reaches EOL.
+
+General Guidelines
+------------------
+
+When referencing a particular release of a product:
+
+* Prefer version directives wherever possible, as this is the easiest target
+  to later find and evaluate:
+
+  .. code-block:: rst
+
+     .. versionadded:: 6.0
+
+  In cases where the reference requires further explanation, add the content 
+  to the directive block:
+
+  .. code-block:: rst
+
+     .. versionchanged:: 4.4
+
+        Starting in MongoDB 4.4, this command returns the following output:
+
+  Snooty supports three version directives: ``versionadded``, 
+  ``versionchanged``, and ``deprecated``
+
+  .. note::
+      
+     Do **not** include multiple version numbers in this directive, as it
+     does not render correctly in HTML.
+
+* In cases where you need to refer to a version inline, include the product
+  name before the version number, (e.g. ``MongoDB 5.0``).  This ensures that
+  a search for version numbers will correctly match product and release for
+  checks.
+
+* Do **not** reference versions freeform or with the phrase ``version X.X``
+  as these are much harder to identify programmatically as references to 
+  a specific product version.
+
+Identifying EOL Releases
+------------------------
+
+`versc <https://github.com/kennethdyer/versc>`__ is a tool for finding references
+to EOL releases of products.  It operates similar to grep, performing a 
+series of Regular Expression matches on each line of each file looking for
+reStructuredText version directives and ``PRODUCT X.Y`` pairings. It then
+extracts the version references from lines it finds and evaluates whether these
+are legitimate version references versus references to unknown (read: EOL) releases.
+
+For example:
+
+.. code-block:: console
+
+   $ versc source/core/ | grep 2d.txt
+   2.2   -   source/core/2d.txt:22   <geospatial-legacy>` from MongoDB 2.2 or earlier, *and*
+   2.2   -   source/core/2d.txt:17   <geospatial-legacy>` used in MongoDB 2.2 and earlier.
+
+This output is not exhaustive (versc doesn't parse the reStructuedText, so it 
+isn't reading include files), but does provide a starting point in identifying
+references to EOL releases.
```

Comment:
`This output is not exhaustive because versc doesn't parse the {+rst+}, so it isn't reading included files. However, versc does help you initially identify
references to EOL releases.`

Reply:
I like @corryroot's suggestion here, I might just also consider shortening "versc does help you initially identify" to "versc helps initially identify..."

Reply:
Removed section in favor of grep example.